src/protocol/notarization/README.md

```diff
@@ -1,20 +1,20 @@
 # Notarization Phase
 
-During the Notarization Phase the `Requester`, otherwise referred to as the `User`, and the `Notary` work together to generate an authenticated `Transcript` of a TLS session with a `Server`.
+During the Notarization Phase the `User` (otherwise referred to as the `Requester` or `Prover`) and the `Notary` work together to generate an authenticated `Transcript` of a TLS session with a `Server`.
```

Comment:
let's get rid of all mentions of `Requester` altogether. we will not use this word anywhere in the code or in the docs.

Reply:
@themighty1 I left it here, because `Requester` is currently still used in [key_exchange.md](https://github.com/tlsnotary/docs-mdbook/blob/1a8d083dff495191907d6bb2a550c2e5567f9fb3/src/protocol/notarization/key_exchange.md#L4) because uses the notation from Wikipedia.